lib/core/Cas.ts

```diff
@@ -79,4 +92,30 @@ export default class Cas implements ICas {
       throw error;
     }
   }
+
+  /**
+   * Gets the cached service version.
+   */
+  public getCachedServiceVersion(): ServiceVersionModel {    
+    return this.cachedVersionModel;
+  }
+
+  /**
+   * Gets the version information by making the REST API call.
+   */
+  private async tryGetServiceVersion() : Promise<ServiceVersionModel> {
```

Comment:
Design: I was imagining this a `ServiceInfoFetcher` class would be doing the fetching and caching, so that `Cas` and `Blockchain` don't have to know about the implementation details.